pkg/tests/regress_aws_1423_test.go

```diff
@@ -107,6 +107,8 @@ func TestRegressAws1423(t *testing.T) {
 		[]byte{}, /* pulumiSchema */
 	)
 
+	shimv2.SetInstanceStateStrategy(p.ResourcesMap().Get("aws_wafv2_web_acl"), shimv2.CtyInstanceState)
```

Comment:
Do we need to regenerate the test case, or should we not expect a diff below this line?

Reply:
The test case does discriminate the functionality. It fails the test once the changes to undo the previous non-fix made to pulumi-terraform-diff-reader are made. It's incomplete in a sense that it couldn't discriminate the pulumi-terraform-diff-reader non-fix from a real fix.

Reply:
I see